app/views/mods_panel.py

```diff
@@ -1917,9 +1943,10 @@ def recreate_mod_list_and_sort(
             None
```

Comment:
Docstring needs to be fixed to include key

Reply:
Also needs improved description that is more accurate.

Reply:
What do you think about the current docstring? Shall we resolve this?